### PR TITLE
Add playwright Test

### PR DESCRIPTION
**Pull request by cniackz**
### Objective:

Start exploring playwright to complement testcafe and if we like it, migrate to playwright little by little.

### How to manually test:

1. Have `console` running at port `5005` as we normally test
2. Under `playwright/tests` new added folder run:

```
npx playwright test
```

* This will execute `verify-title.spec.ts` which is a TypeScript test that checks just the title to contain `MinIO`
* Expected result is:

```sh
$ npx playwright test

Running 1 test using 1 worker

  ✓  1 example.spec.ts:3:5 › has title (624ms)

  1 passed (1.1s)
```

### Dependencies:

Screenshot 2023-03-07 at 9 33 31 AM

### CODE REVIEW

File: .github/workflows/jobs.yaml [hunk 1]
```diff
@@ -1307,3 +1307,45 @@ jobs:
...
+  playwright:
+    needs:
+      - compile-binary
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
```
Comment (reivaj05) [inline .github/workflows/jobs.yaml:1320]: node-version: ${{ env.NVMRC }}

File: .github/workflows/jobs.yaml [hunk 2]
```diff
@@ -1307,3 +1307,45 @@ jobs:
...
+      run: |
+        (./console server) & (make initialize-permissions)
+
+    - name: Run Playwright tests
+      run: |
+        echo "Run tests under playwright folder only"
+        cd $GITHUB_WORKSPACE/playwright
+        npx playwright test
+    - uses: actions/upload-artifact@v3
+      if: always()
```
Comment (reivaj05) [inline .github/workflows/jobs.yaml:1347]: Do we need to run this step if tests failed?

File: playwright/.github/workflows/playwright.yml [hunk 3]
```diff
@@ -0,0 +1,209 @@
+name: Workflow
```
Comment (reivaj05) [inline playwright/.github/workflows/playwright.yml:1]: Why is this file needed?

File: .github/workflows/jobs.yaml (hunk 1 above)
Comment (dvaldivia) [reply .github/workflows/jobs.yaml:1320]: the playwrigth create script proposes this and it uses 16 for the framework

File: playwright/.github/workflows/playwright.yml (hunk 3 above)
Comment (cniackz) [reply playwright/.github/workflows/playwright.yml:1]: It is needed because we can use it to run only playwright in github actions, mainly for testing but can also be removed but manual intervention will be needed to run standalone mode.... Anyway I will soon close this PR because we have a better approach in other PR.

**Comment by cniackz** [comment]
Thank you @reivaj05 for the review, but we are having new approach in https://github.com/minio/console/pull/2702, please help us review it there.
